src/vuex/modules/distribution.js

```diff
@@ -87,10 +89,12 @@ export default ({ node }) => {
       }
     },
     async postMsgWithdrawDelegationReward({ dispatch }) {
-      await dispatch(`getTotalRewards`)
-      await dispatch(`getRewardsFromMyValidators`)
-      await dispatch(`queryWalletBalances`)
-      await dispatch(`getAllTxs`)
+      return Promise.all([
```

Comment:
why do promises work better than `async` `await`?

Reply:
`await`s will wait for each `dispatch` to have completed. `Promise.all` fires all `dispatch`es in parallel and waits for them all to complete.